### PR TITLE
Release/prepare 5.8.0

### PR DESCRIPTION
**Pull request by DaedalusG**
### Test plan
review

### CODE REVIEW

**Review by jdpleiness — APPROVED** [review]

**Review by Chickensoupwithrice — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ The [single-executor example](https://github.com/sourcegraph/terraform-aws-execu
   - 4.2.0 and above: `>= 1.1.0, < 2.0.0`
 - [hashicorp/aws](https://registry.terraform.io/providers/hashicorp/aws) 
   - 4.1.0 and below: `~> 3.0.0`
-  - 4.2.0 and above: `>= 3.0, < 5.6.0`
+  - 4.2.0 and above: `>= 3.0, < 5.8.0`
```
Comment (eseliger) [inline README.md:31]: is this update correct? 🤔

File: README.md [hunk 2]
```diff
@@ -6,19 +6,19 @@ This repository provides a [Terraform module](https://learn.hashicorp.com/tutori
 
 This repository provides four submodules:
```
Comment (eseliger) [inline README.md:8]: strange we went from 5.6 to 5.8 - we missed this in 5.7? can we make sure to add this step somewhere so we don't forget again? :)

**Comment by eseliger** [comment]
make sure to create the 5.8.0 tag :)

File: README.md (hunk 2 above)
Comment (DaedalusG) [reply README.md:8]: Its in our run book? Not sure why it was missed or how we patched the thing

Comment (DaedalusG) [reply README.md:8]: Sounds like we may have just forgot to merge the change

**Pull request merged**